membership/tests.py

```diff
@@ -1076,6 +1076,37 @@ def test_duplicate_payment(self):
         self.assertEqual(cycle.reference_number, payment.reference_number)
         self.assertTrue(cycle.is_paid)
 
+    def test_duplicate_transaction_id(self):
+        no_cycle_q = Q(billingcycle=None)
+
+        with open(data_file("procountor-csv-duplicate-transaction.csv"), 'r') as f:
+            process_procountor_csv(f)
+        payment_match_count = Payment.objects.filter(~no_cycle_q).count()
+        error = "Both payments in the sample file should have matched"
+        self.assertEqual(payment_match_count, 1, error)
+        self.assertEqual(Payment.objects.count(), 2)
+        payment = Payment.objects.filter(billingcycle=self.cycle).latest("payment_day")
+        cycle = BillingCycle.objects.get(pk=self.cycle.pk)
+        self.assertEqual(cycle.reference_number, payment.reference_number)
+        self.assertTrue(cycle.is_paid)
+
+    def test_duplicate_import(self):
+        no_cycle_q = Q(billingcycle=None)
+
+        with open(data_file("procountor-csv-single.csv"), 'r') as f:
+            process_procountor_csv(f)
+        payment_match_count = Payment.objects.filter(~no_cycle_q).count()
+        error = "The sample file should have contained only one payment"
+        self.assertEqual(Payment.objects.count(), 1, error)
+        with open(data_file("procountor-csv-single.csv"), 'r') as f:
+            process_procountor_csv(f)
```

Comment:
What's the reason for doing this twice on the same input file? Should this be two tests?